plugins/osdn/multitenant/multitenant.go

```diff
@@ -31,6 +31,7 @@ func Master(osClient *osclient.Client, kClient *kclient.Client, clusterNetwork s
 	if err != nil {
 		glog.Fatalf("SDN initialization failed: %v", err)
 	}
+	kc.AdminNamespaces = append(kc.AdminNamespaces, "default")
```

Comment:
With multitenant sdn plugin, each namespace will have their own virutal network expect default namespace. Default namespace will have network access to all other namespaces and vice versa. This will enable router, load balancer, etc. to work as expected. This behavior is already shipped in our previous release. This is broken recently and this pull request will restore the intended behavior.
We don't want to expose this behavior via config (like in #4370 ). Instead there may be some oadm/oc command to modify network associated with namespace. Requirement is to enable multiple namespaces to share/unshare some network, mark specific namespace as global/admin. This needs more discussions before we come up with final interface to the end user (https://trello.com/c/5maz2tmx/84-flexible-network-security-policy-network-isolation).